srcpkgs/write-styluslabs/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'write-styluslabs'
+pkgname=write-styluslabs
+version=3.0.0
+revision=1
+archs="x86_64"
+wrksrc="Write"
+depends="SDL2"
+short_desc="Word processor for handwriting"
+maintainer="Cat Marin <me@jmarin.xyz>"
+license="custom:Proprietary"
+homepage="http://www.styluslabs.com/"
+distfiles="http://www.styluslabs.com/write/write-latest.tar.gz"
```

Comment:
The `distfiles` URL is problematic because there is no fixed versioning. Whenever Stylus Labs releases a new version, that link will break the package, so things like package revbumps or local building become impossible. I suggest you try to find a versioned download, or else reach out to Stylus Labs about providing versioned downloads.

Reply:
As far as I can tell that's how they release packages. I actually tried doing it with a variable and xbps-src doesn't accept `version=latest`. Until they provide versioned downloads, I'm not sure how else to proceed.

Reply:
They provides versioned tarball, they just don't show you, here is the link for 3.0.0 http://www.styluslabs.com/write/write300.tar.gz

Reply:
> They provides versioned tarball, they just don't show you, here is the link for 3.0.0 http://www.styluslabs.com/write/write300.tar.gz

Yep! Saw that in their response to me email. However they still don't have a license available, which as I understand is necessary in order to proceed. In any case, judging by how many changes I needed to make (and quality concerns of my template) I'm in no way ready to take on maintaining a package. I'll look more into the documentation and other package templates before I do anything else.

Reply:
Nah, packages in void aren't that hard. TBH, it's really just that you chose a slightly problematic package to start on :)